docs/articles/baseplatform/prerequisites.rst

```diff
@@ -7,9 +7,7 @@ Yocto requires the following packages to be installed in order to build an image
 
 .. code-block:: bash
 
-    git sed wget cvs subversion git-core coreutils unzip gawk python-pysqlite2
-    diffstat help2man make gcc build-essential g++ chrpath libxml2-utils
-    libsdl1.2-dev texinfo python3 gcc-multilib g++-multilib
+    $ apt-get install git sed wget cvs subversion git-core coreutils unzip gawk python-pysqlite2 diffstat help2man make gcc build-essential g++ chrpath libxml2-utils libsdl1.2-dev texinfo python3 gcc-multilib g++-multilib
```

Comment:
I think the purpose of these lines was just to list the dependencies, not to give the command for installing them

Reply:
Check the code block for debian. Its listed as command (apt-get install), yet seperated in 3 lines. I can fix it either way as long as they are consistent but this made more sense.

Reply:
I see what you mean. The thing is the list is very Debian/Ubuntu specific, since in packages names may be different for other distros.

Reply:
@fhajredini as long as it looks nice on the page, either way, is fine.

Reply:
Yeap, im doing it now.